### ISSUE TITLE
SV VCF update?

### ISSUE THREAD

**Issue by lgmgeo** [ChildrensMercyResearchInstitute/GA4K#3]
Dear authors,

Your publicly available "pb_joint_merged.sv.vcf.gz" file is integrated into [AnnotSV](https://lbgi.fr/AnnotSV/) (v3.1 and higher), a program to annotate and rank human Structural Variations.

All that frequency data is really useful.

I would like to know if you have planned an update of this file in a near future or not?

Thank you,

Véronique

**Comment by maximilianh** [comment]
Yes, I have the same question, there is a new GA4K data release, can you also update the file here in this repo?

**Comment by carlschreck** [comment]
Thank you for your interest, and my apologies about the delay! We are hoping to update this file later this year. However, due to computational limitations in adding additional samples via Jasmine, we will be exploring other options for merging. Additionally, PacBio has replaced caller PBSV with sawfish, so we are planning to use sawfish for our updated dataset.

**Comment by maximilianh** [comment]
Jasmine has tons of options to run faster, I think I ran it on more than
1000 samples.  I managed to get it to run by removing all SNVs, remove the
sequence comparison to only edit distance and I found a few other
options... I can dig if that helps. Removing the SNVs solved the problem
for me.

**Comment by maximilianh** [comment]
Carl, I'm trying to send you a note, but I cannot find your email anywhere,
the yale email gets returned, your CMH email isn't anywhere to be found.
can you contact me at ***@***.*** ? thanks.
